### PR TITLE
sv-98-cleanup

### LINKED ISSUE #98: chore(f4 ): main-nav
## Changes
1. Delete navbar component which is not used anymore
2. Rename tagnames for mat-menu items matching to their component names
3. Adjust some css to sidenav

## Purpose
Cleaning and Organizing

## Approach
Readability to teammates

Closes #98

### PR DESCRIPTION
**Pull request by svoggu**
Changes
Delete navbar component which is not used anymore
Rename tagnames for mat-menu items matching to their component names
Adjust some css to sidenav
Purpose
Cleaning and Organizing

Approach
Readability to teammates

Closes #98

### CODE REVIEW

**Review by ChelleAmon — APPROVED** [review]
Good job removing most of the commented codes here! I also love the header. It is the first thing I saw when I open the project. 

For the font color on the sidebar, I am not a fan of this blue type color, but it is just me and I cannot think of a color that works best with this lol.

Overall, great job!

File: src/client/src/app/store/reducers/user/user.reducer.ts [hunk 1]
```diff
@@ -35,6 +35,11 @@ export const reducer = createReducer(
     const users = [...state.users];
     users.push(action.data);
     return {...state, users}
-  })
+  }),
+
+  on(logoutUserSuccess, (state, action) => {
+    return {...state, loginUser: null}
+  }),
+
```
Comment (Aman-codde) [inline src/client/src/app/store/reducers/user/user.reducer.ts:43]: The code is working properly, but I think this file does not belong to this particular issue and pull request, it could have done in another pull request by creating new issue for handling logout.

**Review by Aman-codde — APPROVED** [review]
The responsive navbar looks really good and mobile version will now be more clean. Looks good!

File: src/client/src/app/store/reducers/user/user.reducer.ts (hunk 1 above)
Comment (ChelleAmon) [reply src/client/src/app/store/reducers/user/user.reducer.ts:43]: I agree with Aman on this part since this PR is all about cleaning up some navbar components. Good catch btw.

Comment (svoggu) [reply src/client/src/app/store/reducers/user/user.reducer.ts:43]: Yes Aman, but i want to look at how logout work on main-nav at that time.I think I have to work on more with logout still and will do in seperate PR.
